tests/HasMediaConversionsTrait/DeleteMediaTest.php

```diff
@@ -38,6 +38,17 @@ public function it_can_clear_a_collection()
         $this->assertCount(0, $this->testModelWithoutMediaConversions->getMedia('images'));
     }
 
+    /** @test */
+    public function it_can_clear_a_collection_excluding_a_single_media()
```

Comment:
Here we've tested that the excluded media is not deleted, but this test should assert that the rest of the media is in fact being deleted. Just to be sure.